music21/abcFormat/__init__.py

```diff
@@ -1811,9 +1835,15 @@ def tokenize(self, strSrc):
         collect = []
         lastIndex = len(strSrc) - 1
         skipAhead = 0
+        accidentalsAndDecorations = '.~^=_HLMOPSTuv'
+        accidentals = '^=_'
 
         activeChordSymbol = '' # accumulate, then prepend
-
+        accidentalized = dict()
+        accidental = None
+        pit = None
```

Comment:
what is `pit` here?  We try to avoid abbreviations that go too far.  Is it `pitchStr` or `abcPitch` or `lastPitch`?

Reply:
I went with `abcPitch` (the abc that defines a pitch if present in the token being identified)